src/components/BuildStatus/__tests__/BuildStatus.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { render } from '@testing-library/react';
+import { BuildStatusType } from '../../../scripts/Build';
+import { BrowserRouter } from 'react-router-dom';
+import { BuildStatus } from '../BuildStatus';
+
+test('renders BuildStatus', () => {
```

Comment:
Try to extract the mock data into separate JSON data so that it will be easier to modify the data without touching the test code. see https://github.com/DnsZhou/pnc-web-ui-react/blob/NCL-6755/src/components/ProjectsList/__tests__/ProjectsList.test.tsx#L10-L13